ios/Classes/core/PMManager.m

```diff
@@ -440,15 +440,22 @@ - (void)fetchThumb:(PHAsset *)asset option:(PMThumbLoadOption *)option resultHan
                       contentMode:option.contentMode
                           options:requestOptions
                     resultHandler:^(PMImage *result, NSDictionary *info) {
-        BOOL downloadFinished = [PMManager isDownloadFinish:info];
+        if ([handler isReplied]) {
+            return;
+        }
```

Comment:
This is dead code, no?

Reply:
I have no idea. I didn't want to refactor your code too much. I just moved it to the top since I didn't want `[handler reply: nil];` to be called more than once. But I can remove that if you think it's a dead code.